sys/crypto/modes/ecb.c

```diff
@@ -23,7 +23,7 @@
 
 #include "crypto/modes/ecb.h"
 
-int cipher_encrypt_ecb(cipher_t *cipher, uint8_t *input,
+int cipher_encrypt_ecb(const cipher_t *cipher, uint8_t *input,
```

Comment:
```suggestion
int cipher_encrypt_ecb(const cipher_t *cipher, const uint8_t *input,
```

while you are at it :)